vignettes/datasets.Rmd

```diff
@@ -26,8 +26,10 @@ To create new datasets, multiple paths exist. In the web application, you can up
 
 You can create a dataset from any `data.frame` you have in your R session with `newDataset`:
 
-```{r, eval=FALSE}
-load("economist.RData")
+This package provides one as `economist` which loads a `data.frame` named `df` because old habits die hard.
+
+```{r}
+data("economist", package = "crunch") # named df
```

Comment:
I see below that vignettes/getting-started.RData is modified in this PR, but the file is generated from https://github.com/Crunch-io/rcrunch/blob/master/vignette-data/1-getting-started.R#L24, so any changes to it need to have a corresponding change in vignette-data/1-getting-started.R.